aiohttp_jwt/middleware.py

```diff
@@ -75,6 +76,7 @@ def JWTMiddleware(
                         token,
                         secret_or_pub_key,
                         algorithms=algorithms,
+                        **kwargs
```

Comment:
I doubt that passing kwargs directly is a good approach. Ideally, end-user should not know about the underlying implementation of jwt. So introducing additional properties in `JWTMiddleware` and passing them to jwt.decode will be better as for me.

Reply:
Thanks for comment. I fix it soon.

Reply:
Great, thanks 🙌

Reply:
It was fixed.